### PR TITLE
pdftk-builder: Add version 3.10.0

### PR DESCRIPTION
**Pull request by wenmin92**
[PDFTK Builder](http://www.angusj.com/pdftkb/) is a free graphical interface to PDFTK.

### CODE REVIEW

File: bucket/pdftk-builder.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "http://www.angusj.com/pdftkb/",
+    "description": "A free graphical interface to PDFTK. Take control of your PDF documents - merge or split pages; stamp pages or add background watermarks; add page numbers; and even rotate pages.",
+    "version": "3.10.0",
+    "license": "GPL-3.0-only",
+    "url": "http://www.angusj.com/pdftkb/pdftkb_setup.exe",
+    "hash": "c6eacc1e93185d2e8608dbae643267c893108b699dfef22bcda885b484fafc86",
+    "innosetup": true,
+    "pre_install": [
```
Comment (Ash258) [inline bucket/pdftk-builder.json]: If you run checkver/formatjson as last thing before posting PR you will have this conversions automatically.

```suggestion
    "pre_install": "if (!(Test-Path \"$persist_dir\\PDFTKBuilder.ini\")) { New-Item \"$dir\\PDFTKBuilder.ini\" | Out-Null }",
```

File: bucket/pdftk-builder.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "if (!(Test-Path \"$persist_dir\\PDFTKBuilder.ini\")) { New-Item \"$dir\\PDFTKBuilder.ini\" | Out-Null }"
+    ],
+    "shortcuts": [
+        [
+            "PDFTKBuilder.exe",
+            "PDFTK Builder"
+        ]
+    ],
+    "persist": [
```
Comment (Ash258) [inline bucket/pdftk-builder.json]:
```suggestion
    "persist": "PDFTKBuilder.ini",
```

File: bucket/pdftk-builder.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "homepage": "http://www.angusj.com/pdftkb/",
+    "description": "A free graphical interface to PDFTK. Take control of your PDF documents - merge or split pages; stamp pages or add background watermarks; add page numbers; and even rotate pages.",
```
Comment (Ash258) [inline bucket/pdftk-builder.json]:
```suggestion
    "description": "Graphical interface to PDF ToolKit for document manipulation (merge, split, ...)",
```

File: bucket/pdftk-builder.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "url": "http://www.angusj.com/pdftkb/pdftkb_setup.exe",
+    "hash": "c6eacc1e93185d2e8608dbae643267c893108b699dfef22bcda885b484fafc86",
+    "innosetup": true,
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\PDFTKBuilder.ini\")) { New-Item \"$dir\\PDFTKBuilder.ini\" | Out-Null }"
+    ],
+    "shortcuts": [
+        [
+            "PDFTKBuilder.exe",
+            "PDFTK Builder"
```
Comment (Ash258) [inline bucket/pdftk-builder.json]: Shortcut with this name will be created by installer?

Comment (wenmin92) [reply bucket/pdftk-builder.json]: Yes, the name is splited by a space.

Comment (Ash258) [reply bucket/pdftk-builder.json]: https://github.com/lukesampson/scoop-extras/pull/3056#discussion_r338548603

**Pull request merged**